src/AnnotationThread.js

```diff
@@ -512,9 +514,8 @@ class AnnotationThread extends EventEmitter {
             }
 
             // Remove temporary annotation and replace it with the saved annotation
-            this.dialog.addAnnotation(savedAnnotation);
-            this.dialog.removeAnnotation(tempAnnotation.annotationID);
-            this.dialog.enable(savedAnnotation.annotationID);
+            this.dialog.show(this.annotations);
+            this.dialog.enable(savedAnnotation.id);
```

Comment:
Where did this property get changed from annotationID to id?

Reply:
whoops part of a separate PR!

Reply:
I'm also removing this method call altogether in #220